src/pages/balance/components/Activity.tsx

```diff
@@ -117,85 +132,126 @@ export const Activity = () => {
       .sort((a, b) => (b.lastUpdated as number) - (a.lastUpdated as number));
   }, [search, activity]);
 
-  const allChains = chains.flatMap((c) => c.chains);
+
+
 
   if (isLoading) {
     return <Loading />;
   }
+  
 
   return (
     <>
-      <input
-        data-testid="search-input"
-        id="search"
-        placeholder={t("search") as string}
-        className="input-primary"
-        onChange={(e) => {
-          setSearch(e.target.value);
-        }}
-      />
-
-      <div className="flex flex-col my-5 overflow-y-auto h-full">
+      <div className={stylesActivity.containerGlobal}>
+          <input
+            data-testid="search-input"
+            id="search"
+            placeholder={t("search") as string}
+            className={stylesActivity.inputTxSearch}
+            onChange={(e) => {
+              setSearch(e.target.value);
+            }}
+          />
+          <CiSearch className={stylesActivity.iconSearch} />
+      </div>
+      <div className={stylesActivity.countainerTx}>
         {activity.length === 0 && (
-          <div className="flex justify-center items-center mt-5">
-            <p className="text-lg font-medium">{t("empty")}</p>
+          <div className={` ${stylesActivity.flexItemsCenter}${stylesActivity.countainerEmptyActivity}`}>
```

Comment:
This is generating a className like this:

```
className="flex items-centerjustify-center mt-5"
```

add a space between this twos classes:
```
`${stylesActivity.flexItemsCenter} ${stylesActivity.countainerEmptyActivity}`
```

Reply:
done